src/templates/_common/components/loading.jsx

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
 
+/*
+*   TODO: Remove the extra classes when splitting the project.
+*   Binary 1.0 and Binary Nex is using different classNames
+*   to accomplish the same loader effect.
+*
+*   Classes for Binary 1.0 :
+*       <div className={barspinner dark}
+*           <div className={rect${inx+1}}
+*
+*   Classes for Binary Nex :
+*       <div className={barspinner barspinner--${ theme || 'dark'}}
+*           <div className={barspinner__rect barspinner__rect--${inx + 1}}
+*
+*   As a temporary solution, we will add both classes for 1.0 and Nex,
+*   however we should remove it in the future.
+*/
+
 const Loading = ({ is_invisible, theme, id }) => (
-    <div id={id} className={`barspinner barspinner--${ theme || 'dark'}${is_invisible ? ' invisible' : ''}`}>
```

Comment:
It's a different implementation for Binary 2.0. See the `loading.scss` file in `app_2`. It's pretty much a copy paste of the `barspinner` class from binary style but with `bem` implemented, therefore we should keep it like this otherwise the loader animation in binary 2.0 will not work.

Reply:
IMO it's not a common component anymore if we have different classes and different styling files for it, so we should split it and keep a copy of each in app and app_2 folders with the needed classes.